lib/preview-frame.ts

```diff
@@ -21,6 +25,24 @@ function loadP5(version: string, cb?: () => void) {
   document.body.appendChild(script);
 }
 
+function loadScripts(urls: string[], cb?: () => void) {
```

Comment:
:+1: you could get fancy with promises, but looping over the scripts and loading each should get the job done